hawkular-alerts-engine/pom.xml

```diff
@@ -88,6 +88,12 @@
       <version>${version.com.google.guava}</version>
     </dependency>
 
+    <dependency>
+      <groupId>io.reactivex</groupId>
+      <artifactId>rxjava</artifactId>
```

Comment:
@lucasponce I don't have anything against RxJava, and I thought perhaps it was something to consider for the provider/consumer scenario.  This in no way indicates any plan for re-write of existing, working code.   I was just wondering if it's a good solution to *this* problem.  I'm not so sure I understand the "mixed technology" thing, it's just a java API.